### DISCUSSION TITLE
How to make different production scheduling rules?

### DISCUSSION THREAD

**Discussion by theskysea** [frePPLe/frepple#540] [Q&A]
The production schedule is inverted according to the delivery date in the frepple.
How to set the positive production sequence?

**Comment by jdetaeye** [comment]
The frePPLe algorithm doesn't use "scheduling rules", aka "dispatching rules".  The planning algorithm is a heuristic search that sorts demands by priority and due, and then searches the supply path for each demand to reserve material and capacity.

There are many reasons why you could see an "inverted sequence" in the resulting plan...    So, I can't comment on your specifics.
One configuration that wouldn't have such inversions is an ASAP plan where all demands are planned ASAP in forward scheduling mode. To achieve that, just set the parameter "plan.administrativeleadtime" to a high value like 365.

**Reply by theskysea** [reply]
thank you
